packages/shared-mocks-lib/src/repo/settlement_batch_transfer_repo_mock.ts

```diff
@@ -45,8 +47,32 @@ export class SettlementBatchTransferRepoMock implements ISettlementBatchTransfer
 		return Promise.resolve();
 	}
 
-	async getBatchTransfersByBatchIds(batchIds: string[]): Promise<ISettlementBatchTransfer[]>{
-		return this._list.filter(value => batchIds.includes(value.batchId));
+	async getBatchTransfersByBatchIds(
```

Comment:
In these methods, because we call them in many places in the aggragte, we need the default to be return all (without pagination)